.github/workflows/ci-docs-linkcheck.yml

```diff
@@ -0,0 +1,96 @@
+# reference:
+#   - https://github.com/actions/cache
+#   - https://github.com/actions/checkout
+#   - https://github.com/marketplace/actions/setup-miniconda
+
+name: ci-docs-linkcheck
+
+on:
+  push:
+    branches:
+      - "*" 
+
+  pull_request:
+    branches:
+      - "*" 
+
+jobs:
+  tests:
+    name: ${{ matrix.os }} ${{ matrix.python-version }}
+
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.8"]
+        include:
+          - os: ubuntu-latest
+            lock-file: requirements/ci/nox.lock/py38-linux-64.lock
+
+    env:
+      ENV_NAME: "iris-ci"
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Conda package cache"
+        uses: actions/cache@v2
+        env:
+          # Increment the build number to force a cache refresh.
+          CACHE_BUILD: 0
+          LOCK_FILE: ${{ matrix.lock-file }}
+        with:
+          path: ~/conda_pkgs_dir
+          key: ${{ runner.os }}-conda-pkgs-py${{ matrix.python-version }}-b${{ env.CACHE_BUILD }}-${{ hashFiles(env.LOCK_FILE) }}
+
+      - name: "Conda install"
```

Comment:
TBH I took that route for convenience, so that we could have total control and opt into everything that [setup-miniconda](https://github.com/conda-incubator/setup-miniconda) offers, such as easily switching between `Miniforge3` and `Mambaforge`.

I guess it depends on peoples appetite for that...